src/mpyl/steps/deploy/bpm_deploy.py

```diff
@@ -29,17 +29,17 @@ def __init__(self, logger: Logger) -> None:
     def execute(self, step_input: Input) -> Output:
         bpm_deploy_results = []
         camunda_config = CamundaConfig.from_config(
-            step_input.run_properties.config,
-            step_input.run_properties.target,
+            step_input.run_properties,
             step_input.project_execution.project,
         )
 
         project_name = step_input.project_execution.project.name
+        pr_number = camunda_config.pr_number
         docker_result = deploy_to_cluster(self._logger, project_name, camunda_config)
         bpm_deploy_results.append(docker_result)
         if docker_result.success:
             modeler_result = deploy_to_modeler(
-                self._logger, project_name, camunda_config
+                self._logger, project_name, camunda_config, pr_number
```

Comment:
you already passed camunda_config which already contains pr_number so I don't think you need to pass it as an argument